reference/cric.v2.yaml

```diff
@@ -1047,6 +1047,213 @@ paths:
         - autorizacao
       description: Recover user's password.
     parameters: []
+  /collections:
+    get:
+      summary: List Collection
+      tags: []
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/CollectionModel'
+        '400':
+          description: Bad Request
+        '401':
+          description: Unauthorized
+        '403':
+          description: Forbidden
+        '404':
+          description: Not Found
+      operationId: get-collections
+      description: Get all collections.
+      parameters:
+        - schema:
+            type: integer
+          in: query
+          name: id_usuario
+    post:
+      summary: New Collection
+      operationId: post-collections
+      responses:
+        '201':
+          description: Created
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/CollectionModel'
+        '400':
+          description: Bad Request
+        '401':
+          description: Unauthorized
+        '403':
+          description: Forbidden
+      description: Create a single collection.
+      security:
+        - API Key Authorization: []
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                name:
+                  type: string
+                description:
+                  type: string
+                slang:
+                  type: string
+                owner:
+                  type: integer
+        description: ''
+  '/collections/{id_collection}':
+    parameters:
+      - schema:
+          type: string
+        name: id_collection
+        in: path
+        required: true
+    get:
+      summary: Your GET endpoint
+      tags: []
+      responses:
+        '200':
+          description: OK
+        '400':
```

Comment:
We don't have 400. If the user provide a string instead of an integer the end point would not match.